### DISCUSSION TITLE
Need to accept multiple ports for SSL (HTTPProxy)

### DISCUSSION THREAD

**Discussion by jacobpoly73** [projectcontour/contour#5669] [General]
I have a situation where my HTTPProxy needs to accept requests on 2 ports. Lets say 443 is the default...but then I also want to listed on 8443. Is this doable? OR is it possible to redirect the 8443 port to 443 at the HTTPProxy level? Currently my yaml is as follows

apiVersion: projectcontour.io/v1
kind: HTTPProxy
metadata:
  name: my-httpproxy
  namespace: my-ns
spec:
  routes:
  - conditions:
    - prefix: /
    services:
    - name: nginx
      port: 8000
  virtualhost:
    fqdn: myurl.com
    tls:
      secretName: my-tls

How can I add support for 8443 in addition to 443. OR redirect all 8443 to 443?

Thanks in advance!

**Comment by skriss** [comment]
@jacobpoly73 this is not currently possible with HTTPProxy, however with the upcoming Contour 1.26, if you are using Gateway API/the Gateway provisioner, you'll be able to define an arbitrary number of TLS ports -- see https://github.com/projectcontour/contour/blob/release-1.26.0-rc.1/changelogs/CHANGELOG-v1.26.0-rc.1.md#support-for-gateway-listeners-on-more-than-two-ports
